scout/ecm_prep_args.py

```diff
@@ -91,9 +89,8 @@ def translate_inputs(opts: argparse.NameSpace) -> argparse.NameSpace:  # noqa: F
         else:
             warn_text = "sector-level 8760 load shapes"
         warnings.warn(
-            "WARNING: Analysis regions were set to EMM to allow " +
-            warn_text + ": ensure that ECM data reflect these EMM regions "
-            "(and not the default AIA regions)")
+            f"WARNING: Analysis regions were set to EMM to allow {warn_text}; "  # noqa: E702
```

Comment:
Note I am explicitly ignoring error E702 here ("E702 multiple statements on one line (semicolon)"). I have run into this issue a few times where flake8 is incorrectly interpreting text as code (only on CI strangely) and throwing an error, but couldn't find a workaround outside of this one.